drivers/node/ssh/ssh.go

```diff
@@ -178,66 +216,88 @@ func (s *ssh) doCmd(addr string, cmd string, ignoreErr bool) error {
 	}
 
 	if err := session.RequestPty("xterm", 80, 40, modes); err != nil {
-		return &ErrFailedToRunCommand{
+		return "", &ErrFailedToRunCommand{
 			Addr:  addr,
 			Cause: fmt.Sprintf("request for pseudo terminal failed: %s", err),
 		}
 	}
+	logrus.Infof("Terminal modes set here")
 
 	stdout, err := session.StdoutPipe()
 	if err != nil {
-		return &ErrFailedToRunCommand{
+		return "", &ErrFailedToRunCommand{
 			Addr:  addr,
 			Cause: fmt.Sprintf("Unable to setup stdout for session: %v", err),
 		}
 	}
+	logrus.Infof("STDOUT set here")
 
 	chOut := make(chan string)
 	go func() {
 		var bufout bytes.Buffer
 		io.Copy(&bufout, stdout)
 		chOut <- bufout.String()
 	}()
-
+	logrus.Infof("go func loop here")
 	stderr, err := session.StderrPipe()
 	if err != nil {
-		return &ErrFailedToRunCommand{
+		return "", &ErrFailedToRunCommand{
 			Addr:  addr,
 			Cause: fmt.Sprintf("Unable to setup stderr for session: %v", err),
 		}
 	}
-
+	logrus.Infof("STDERR set here")
 	chErr := make(chan string)
 	go func() {
 		var buferr bytes.Buffer
 		io.Copy(&buferr, stderr)
 		chErr <- buferr.String()
 	}()
-
-	if err = session.Run(cmd); !ignoreErr && err != nil {
-		return &ErrFailedToRunCommand{
+	logrus.Infof("go func2 loop here")
+	byteout, err := session.CombinedOutput(cmd)
+	out = fmt.Sprintf("%v", byteout)
+	if ignoreErr == false && err != nil {
+		return out, &ErrFailedToRunCommand{
 			Addr:  addr,
 			Cause: fmt.Sprintf("failed to run command due to: %v", err),
 		}
 	}
-
-	return nil
+	logrus.Infof(fmt.Sprintf("Command %s has output %s", cmd, out))
```

Comment:
No need to print this. We should only print errors/warnings and important events in the test. Not traces.